catalogapp/log4j/log4j.properties

```diff
@@ -1,11 +1,8 @@
-log4j.rootLogger=WARN, R
+log4j.rootLogger=WARN, S
 
-log4j.logger.catalogapp=WARN, R
+log4j.logger.org.georchestra.catalogapp=WARN, S
 
-log4j.appender.R = org.apache.log4j.rolling.RollingFileAppender
-log4j.appender.R.RollingPolicy = org.apache.log4j.rolling.TimeBasedRollingPolicy
-log4j.appender.R.RollingPolicy.FileNamePattern = /tmp/catalogapp.%d.log.gz
-log4j.appender.R.RollingPolicy.ActiveFileName = /tmp/catalogapp.log
-log4j.appender.R.Append = true
-log4j.appender.R.layout = org.apache.log4j.PatternLayout
-log4j.appender.R.layout.ConversionPattern = %d{yyyy-MM-dd HH:mm:ss} %c{1} [%p] %m%n
+log4j.appender.S = org.apache.log4j.ConsoleAppender
+log4j.appender.S.Append = true
```

Comment:
`Append` parameter is only needed for files appenders
see http://logging.apache.org/log4j/2.x/manual/appenders.html#ConsoleAppender